scripts/gdb.py

```diff
@@ -43,6 +44,20 @@ def find_vma_base(addr):
         return start-offset
   return None
 
+def get_text_offset(file_name):
+  #command=['readelf', '-SW', file_name]
+  command="readelf -SW {}".format(file_name)
+  proc = Popen(command, stdout=PIPE, stderr=PIPE, shell=True)
+  sections, err = proc.communicate()
+  if proc.returncode != 0:
+      raise Exception("ELF Read Error")
+  for line in sections.decode('utf-8').rstrip().split('\n'):
+      if ".text" in line:
+          text_info = line.split()
```

Comment:
Split first, then check if `text_info[2] == ".text"`

Reply:
might have changed with regular expressions, please let me know after I push the code.